### PR TITLE
RELEASE: 1.3.2

### PR DESCRIPTION
**Pull request by Krzmbrzl**
This commit updates the respective release information and also creates
a new blog entry for it.

### CODE REVIEW

File: hugo/content/blog/2020-07-09-mumble-1-3-2-release.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: Mumble 1.3.2 Release Announcement
+author: Robert Adam
+date: 2020-07-09
+categories:
+  - News
+  - Release
+  - Security
```
Comment (Kissaki) [inline hugo/content/blog/2020-07-09-mumble-1-3-2-release.md:8]: What makes this a security release? I think that’s wrong?

File: hugo/content/blog/2020-07-09-mumble-1-3-2-release.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+title: Mumble 1.3.2 Release Announcement
+author: Robert Adam
+date: 2020-07-09
+categories:
+  - News
+  - Release
+  - Security
+---
+The Mumble team has released [**version 1.3.2**](https://github.com/mumble-voip/mumble/releases/tag/1.3.2) of the Mumble VoIP application. This is a
+**bugfix and security release** of the current and stable 1.3 series.
```
Comment (Kissaki) [inline hugo/content/blog/2020-07-09-mumble-1-3-2-release.md:11]: Second reference to it being a security release.

File: hugo/content/blog/2020-07-09-mumble-1-3-2-release.md (hunk 1 above)
Comment (Krzmbrzl) [reply hugo/content/blog/2020-07-09-mumble-1-3-2-release.md:8]: Ah yes. This is just from copy&pasting xD

File: hugo/content/blog/2020-07-09-mumble-1-3-2-release.md (hunk 2 above)
Comment (Krzmbrzl) [reply hugo/content/blog/2020-07-09-mumble-1-3-2-release.md:11]: I thought this is how we call these minor releases. I'll change it though :point_up:

**Pull request merged**